config.json

```diff
@@ -5,7 +5,7 @@
   "execProgram": "node",
   "exposeRoute": false,
   "imgUrl": "logo.png",
-  "version": "2.8.0",
+  "version": "3.0.0",
```

Comment:
Refactors by nature don't introduce new features, so this is a new patch at most - `2.8.1`